### ISSUE TITLE
Don't Require Rubygems

### ISSUE THREAD

**Issue by sts** [dalen/puppetdb-muninplugins#3]
I think you can remove the require for rubygems since both `net/http` and `json` are part of Ruby stdlib.

**Comment by dalen** [comment]
yeah, I think that was for ruby 1.8 compatibility when json was outside stdlib. But hopefully no one still uses ruby 1.8
